javascript/cable_ready.js

```diff
@@ -160,7 +160,7 @@ const DOMOperations = {
     })
     const parent = element.parentElement
     const ordinal = Array.from(parent.children).indexOf(element)
-    morphdom(element, template.content, {
+    morphdom(element, childrenOnly ? template.content : template.innerHTML, {
```

Comment:
FWIW I _think_ `firstElementChild` will have the same net effect as passing `innerHTML`, but will save morphdom from having to [construct an Element from a string](https://github.com/patrick-steele-idem/morphdom/blob/master/src/morphdom.js#L26-L34):

```suggestion
    morphdom(element, childrenOnly ? template.content : template.content.firstElementChild, {
```

cc @hopsoft